releases.yaml

```diff
@@ -16,6 +16,16 @@ lts:
   release_date: "April 2022"
   eol: "April 2027"
   release_notes_url: "https://discourse.ubuntu.com/t/jammy-jellyfish-release-notes/24668"
+previous_latest:
+  slug: ImpishIndri
+  name: "Impish Indri"
+  short_version: "21.10"
+  full_version: "21.10"
+  core_version: "22"
+  release_date: "October 2021"
+  eol: "July 2022"
+  past_eol_date: true
+  release_notes_url: "https://discourse.ubuntu.com/t/impish-indri-release-notes/21951"
```

Comment:
I don't think we need this? It doesn't look like this data is consumed anywhere, it's only LTS's that have historical data.

Reply:
Right. Removed.